lib/pages/chat_list/chat_list.dart

```diff
@@ -603,12 +611,237 @@ class ChatListController extends State<ChatList>
     return Matrix.of(context).activeBundle;
   }
 
-  void resetActiveBundle() {
-    WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
-      setState(() {
-        Matrix.of(context).activeBundle = null;
-      });
-    });
+  void _onTapBottomNavigation(
+    ChatListSelectionActions chatListBottomNavigatorBar,
+  ) async {
+    switch (chatListBottomNavigatorBar) {
+      case ChatListSelectionActions.read:
```

Comment:
this file too big
please move switch case enum to enum extension

Reply:
it's need func inside controller